docs/src/man/guide.md

```diff
@@ -177,61 +166,6 @@ build/
     See the [Hosting Documentation](@ref) section for details regarding how you should go
     about setting this up correctly.
 
-At this point `build/index.html` should be an empty page since `src/index.md` is empty. You
-can try adding some text to `src/index.md` and re-running the `make.jl` file to see the
-changes.
-
-
-## Adding Some Docstrings
-
-Next we'll splice a docstring defined in the `Example` module into the `index.md` file. To
-do this first document a function in that module:
-
-```julia
-module Example
-
-export func
-
-"""
-    func(x)
-
-Returns double the number `x` plus `1`.
-"""
-func(x) = 2x + 1
-
-end
-```
-
-Then in the `src/index.md` file add the following
-
-````markdown
-# Example.jl Documentation
-
-```@docs
-func(x)
-```
-````
-
-When we next run `make.jl` the docstring for `Example.func(x)` should appear in place of
-the `@docs` block in `build/index.md`. Note that *more than one* object can be referenced
-inside a `@docs` block -- just place each one on a separate line.
-
-Note that a `@docs` block is evaluated in the `Main` module. This means that each object
-listed in the block must be visible there. The module can be changed to something else on
-a per-page basis with a `@meta` block as in the following
-
-````markdown
-# Example.jl Documentation
-
-```@meta
-CurrentModule = Example
-```
-
-```@docs
-func(x)
-```
-````
-
 ### Filtering included docstrings
```

Comment:
I think heading should be "promoted" now, since `## Adding Some Docstrings` was removed:
```suggestion
## Filtering included docstrings
```